### ISSUE TITLE
force push does not work correctly with AuraDefinitionBundle (src/aura/ folder)

### ISSUE THREAD

**Issue by yurybond** [heroku/force#374]
The aura folder in salesforce project tree has a structure different from majority of metadata folders
So when I try to push AuraDegfinitionBundle it sends only the first component found
`force.exe push -t AuraDefinitionBundle`

I was not able to do something when trying to use 'aura' group of commands - not sure what the problem. for example I tried to push aura bundle folder using the following command:
`build/force.exe aura push -f src/aura/inputLookup`
and output of the command was
`GetFileAttributesEx C:\Users\ybondarau\sfprojects\atlas_dev\src\aura\inputLookup\inputLookup`

I guess the reason of problem with push somewhere in _findMetadataFolder_ function:

 force/push.go func findMetadataFolder(dir string) (folderPath string)  (line 254)

To sum up: Please update push command to work correctly with src/aura directory and (AuraDefinitionBundle metadata type)

**Comment by yurybond** [comment]
Guys, any news on AuraDefinitionBundle fetching/pushing?
